pkg/odo/cli/pipelines/wizard.go

```diff
@@ -51,31 +55,60 @@ func NewWizardParameters() *WizardParameters {
 // If the prefix provided doesn't have a "-" then one is added, this makes the
 // generated environment names nicer to read.
 func (io *WizardParameters) Complete(name string, cmd *cobra.Command, args []string) error {
+
 	io.GitOpsRepoURL = ui.EnterGitRepo()
+	io.GitOpsRepoURL = utility.AddGitSuffixIfNecessary(io.GitOpsRepoURL)
+	_, err := sc.NewRepository(io.GitOpsRepoURL)
+	if err != nil {
+		return err
+	}
 	option := ui.SelectOptionImageRepository()
 	if option == "Openshift Internal repository" {
 		io.InternalRegistryHostname = ui.EnterInternalRegistry()
 		io.ImageRepo = ui.EnterImageRepoInternalRegistry()
 
 	} else {
 		io.DockerConfigJSONFilename = ui.EnterDockercfg()
+		fs := ioutils.NewFilesystem()
+		_, err := pipelines.CheckFileExists(fs, io.DockerConfigJSONFilename)
+		if err != nil {
+			return err
+		}
 		io.ImageRepo = ui.EnterImageRepoExternalRepository()
```

Comment:
This question asks the user to enter the image repository of the form `<registry>/<username>/<repository>`. Should we validate that?

Reply:
Will be addressed in a new ticket, since external repo for docker can sometimes be of the form <registry>/<image-repo>!